photon-icons.json

```diff
@@ -1923,7 +1923,7 @@
       }
     },
     { "name": "Tool Options",
```

Comment:
I know the svg file has a different name, but is this the same icon as https://design.firefox.com/icons/viewer/#preferences%20mac (`icons/desktop/preferences-16-macos.svg`)? And if so, can we use that file here, too?

Reply:
Also for the dock-undock we could use this icon https://github.com/FirefoxUX/photon-icons/blob/master/icons/desktop/open-in-new-16.svg

Reply:
Yep, let me compare those files tomorrow and merge/rename them as appropriate.

Unfortunately for the dock-undock one we need to match the proportions of the other dock icons so we can't use "Open in New" as-is.